mappings/net/minecraft/util/SystemUtil.mapping

```diff
@@ -21,6 +21,7 @@ CLASS m net/minecraft/util/SystemUtil
 	METHOD a createTranslationKey (Ljava/lang/String;Lqd;)Ljava/lang/String;
 		ARG 0 type
 		ARG 1 id
+	METHOD a tryConsumeOrRun (Ljava/util/Optional;Ljava/util/function/Consumer;Ljava/lang/Runnable;)Ljava/util/Optional;
```

Comment:
I do not like this name.

Reply:
Just drop it for now.

Reply:
FYI in java 9 optional has `ifPresentOrElse`.